targets/targets.json

```diff
@@ -2696,7 +2696,7 @@
         "supported_form_factors": ["ARDUINO"],
         "inherits": ["MCU_NRF52840"],
         "macros_add": ["BOARD_PCA10056", "CONFIG_GPIO_AS_PINRESET", "SWI_DISABLE0", "NRF52_ERRATA_20"],
-        "device_has": ["ANALOGIN", "ERROR_PATTERN", "I2C", "I2C_ASYNCH", "INTERRUPTIN", "LOWPOWERTIMER", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SERIAL_ASYNCH", "SERIAL_FC", "SLEEP", "SPI", "SPI_ASYNCH", "SPISLAVE"],
```

Comment:
We have hardware bug - not so easy to be mitigated (if possible at all).  HW Bug will be fixed in silicon for nRF52840 - as it is still preview mcu revision. After fix will coming it will be very easy to just turn SPI_ASYNCH back by reverting this changes.